cli/commands/route_list.go

```diff
@@ -0,0 +1,255 @@
+// Copyright 2016 IBM Corporation
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+package commands
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/amalgam8/amalgam8/cli/api"
+	"github.com/amalgam8/amalgam8/cli/common"
+	"github.com/amalgam8/amalgam8/cli/terminal"
+	"github.com/amalgam8/amalgam8/cli/utils"
+	"github.com/urfave/cli"
+	"sort"
+	"strings"
+)
+
+// RouteListCommand is used for the route-list command.
+type RouteListCommand struct {
+	ctx        *cli.Context
+	registry   api.RegistryClient
+	controller api.ControllerClient
+	term       terminal.UI
+}
+
+// NewRouteListCommand constructs a new Route List.
+func NewRouteListCommand(term terminal.UI) (cmd *RouteListCommand) {
+	return &RouteListCommand{
+		term: term,
+	}
+}
+
+// GetMetadata returns the metadata.
+func (cmd *RouteListCommand) GetMetadata() cli.Command {
+	T := utils.Language(common.DefaultLanguage)
+	return cli.Command{
+		Name:        T("route_list_name"),
```

Comment:
IIRC, T() allows internationalization. I don't expect command names to be localized.
This applies to all commands...